app/src/util.cc

```diff
@@ -342,5 +342,15 @@ std::vector<std::string> SplitString(const std::string& s,
   return split_parts;
 }
 
+std::string CreateApiIdentifier(const char* api_id, void* object) {
```

Comment:
const void* object, if it's only using the pointer itself

Reply:
So the nice thing about the string, beyond the debugging help, is the whole memory management part of it.  I'm not sure how the best way to have a generic way to return a unique pointer that could then be easily deleted by the caller later without using a string.